csrc/python_frontend/fusion_definition.cpp

```diff
@@ -624,4 +626,12 @@ std::vector<std::pair<double, double>> FusionDefinition::getValTolerances(
   return get_val_constants(preschedFusion(), inputs);
 }
 
+void FusionDefinition::clone(FusionDefinition& other) {
+  NVF_CHECK(id().has_value(), "FusionDefinition definition does not exist!");
```

Comment:
nitpick, `id().has_value()` is just a `complete()` here. I think it looks cleaner to keep it consistent with the `!other.complete()`.